package/boot/uboot-envtools/files/ramips

```diff
@@ -63,6 +63,9 @@ xiaomi,mi-router-ac2100|\
 xiaomi,redmi-router-ac2100)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x20000"
 	;;
+beeline,smartbox-turbo-plus)
```

Comment:
Alphabetic sorting ...

Reply:
Not sure if the sort is correct. Roughly corrected.